scripts/assemble_changelog.py

```diff
@@ -219,12 +219,14 @@ def add_categories_from_text(self, filename, line_offset,
                                        category.name.decode('utf8'))
 
             body_split = category.body.splitlines()
+            line_number = 1
             for line in body_split:
                 if len(line) > MAX_LINE_LENGTH:
                     raise InputFormatError(filename,
-                                           line_offset + category.title_line,
-                                           'Category body line too long: "{} ({})"',
-                                           category.name.decode('utf8'), len(line))
+                                           line_offset + category.title_line + line_number,
```

Comment:
`category.title_line` is not right when the category title line has blank lines after it. This may be a preexisting bug as `body_line` doesn't seem right either. Not a blocker.

Reply:
Indeed, seems like the script is cutting out blank lines.

Reply:
Found it. It helps if you use the correct offset to start with...